### ISSUE TITLE
No EIP191 signing for you! 🍲

### ISSUE THREAD

**Issue by cygnusv** [nucypher/nucypher#1566]
Our current approach to multisig-powered transactions via our `MultiSig` contract assumes that multisig signers sign a proposed transaction following the [EIP191 version 0 standard](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-191.md) (i.e., data for intended validator).

Even though we funded work in that direction (see https://github.com/ethereum/go-ethereum/issues/17696 and its implementation in https://github.com/ethereum/go-ethereum/pull/17789), it didn't land as we expected, as that new endpoint is part of `clef` and they decided to not offer that functionality via geth.

So, basically, we currently can't sign with EIP191 version 0 via geth, and according to clef's maintainers, it will never be, as they plan to deprecate geth's account management, in favour of `clef`.

So....we only have three options here:
1) Change the `MultiSig` contract to use version `E` (i.e., geth's `personal_sign`).
2) Implement a "wallet"-like client that interfaces with `clef`
3) Adiós, `MultiSig` 😢 

Personally, I prefer option 1, as it's simple and we can still manage to go in a different direction later, even with the MultiSig contract deployed.

**Comment by michwill** [comment]
Sounds like option 1 is the best way forward for now

**Comment by cygnusv** [comment]
Closed since we have support to Clef.
